### PR TITLE
[MM-214] Add user's theme support in link tooltip component

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**
#### Summary
- Add user's theme support in link tooltip component

#### Screenshot
image
image
#### What to test?
- Link tooltip component is working properly according to the different themes.

#### Ticket Link:
Fixes: https://community.mattermost.com/core/pl/97pq3s9u6bnd7ghxx8qpu1x7oc

### CODE REVIEW

File: webapp/src/components/link_tooltip/tooltip.css [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 /* Gitlab Tooltip */
 .gitlab-tooltip .box {
-    background-color: white;
-    box-shadow: 0 1px 15px rgba(27,31,35,.15);
+    background-color: var(--center-channel-bg);
+    box-shadow: 0 1px 10px rgba(var(--center-channel-color-rgb), 0.2);
```
Comment (matthewbirtch) [inline webapp/src/components/link_tooltip/tooltip.css:18]: don't need to theme the shadow here. Shadows are always black

```box-shadow: 0px 12px 32px 0px rgba(0, 0, 0, 0.12);```

File: webapp/src/components/link_tooltip/tooltip.css [hunk 2]
```diff
@@ -27,13 +27,14 @@
 
 /* Header */
 .gitlab-tooltip .header {
-    color: var(--light-gray);
+    color: var(--center-channel-color);
```
Comment (matthewbirtch) [inline webapp/src/components/link_tooltip/tooltip.css:30]: ```color: rgba(var(--center-channel-color-rgb), 0.75);```

File: webapp/src/components/link_tooltip/tooltip.css [hunk 3]
```diff
@@ -27,13 +27,14 @@
...
     font-size: 12px;
+    opacity: 0.7;
```
Comment (matthewbirtch) [inline webapp/src/components/link_tooltip/tooltip.css:32]: don't need opacity with the above change

File: webapp/src/components/link_tooltip/tooltip.css [hunk 4]
```diff
@@ -52,11 +53,12 @@
 .gitlab-tooltip .tooltip-info > a, .gitlab-tooltip .tooltip-info > a:hover {
     display: block;
     text-decoration: none;
-    color: var(--dark-gray);
+    color: var(--center-channel-color);
 }
 
 .mr-number {
-    color: var(--light-gray);
+    color: var(--center-channel-color);
```
Comment (matthewbirtch) [inline webapp/src/components/link_tooltip/tooltip.css:60]: ```color: rgba(var(--center-channel-color-rgb), 0.75);```

and remove opacity

**Comment by raghavaggarwal2308** [comment]
@matthewbirtch Fixed the review comments added by you

**Review by ayusht2810 — APPROVED** [review]

**Review by matthewbirtch — APPROVED** [review]

**Review by mickmister — APPROVED** [review]

**Review by AayushChaudhary0001 — APPROVED** [review]
This PR has been tested for all the themes present on MM server. For all the themes, the tooltip was working fine, LGTM. Approved

**Pull request merged**